lib/improver/tests/acceptance/test_apply_emos_coefficients.py

```diff
@@ -47,8 +47,9 @@ def test_gaussian(tmp_path):
     emos_est_dir = kgo_dir / "../../estimate-emos-coefficients"
     emos_est_path = emos_est_dir / "gaussian/kgo.nc"
     output_path = tmp_path / "output.nc"
-    args = [input_path, emos_est_path, output_path,
-            "norm", "--random_seed", "0"]
+    args = [input_path, emos_est_path,
+            "--distribution", "norm", "--random-seed", "0",
+            "--output", output_path]
```

Comment:
Again, just the style point here - where it's a `"--name"` argument it's clearer IMO to have one argument per line.

Reply:
Fair point. Done